<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditCalc</class>
 <widget class="QWidget" name="creditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>credit calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">creditCalc {
	background-color: rgb(51, 255, 102);
}</string>
  </property>
  <widget class="QRadioButton" name="radioButton_ann">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>145</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>annuity</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_diff">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>145</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>differentiated</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Credit type:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_calculate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>calculate</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>217</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Total loan amount:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxTotalCreditSum">
        <property name="maximum">
         <number>100000000</number>
        </property>
        <property name="singleStep">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Term(months):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxTerm">
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Interest Rate(%):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxInterestRate">
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>175</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Monthly payment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_monthly_payment">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Overpayment on the loan:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_overpayment">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Total repayment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total_repayment">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
